### PR TITLE
docs: sealer is not supported by root users

### PR DESCRIPTION
**Pull request by cubxxw**
Signed-off-by: Xinwei Xiong <3293172751nss@gmail.com>

### CODE REVIEW

File: src/docs/getting-started/quick-start.md [hunk 1]
```diff
@@ -98,3 +98,4 @@ sealer delete -a
 + [Kubefile](../concept/kubefile.md)
 + [Sealer Image](../concept/sealer-image.md)
 + [Sealer Cluster Image List](../sealer-images/cluster-images.md)
++ [Running sealer Node Components as a Non-root User](../advanced/use-sealer-rootless.md)
```
Comment (kakaZhou719) [inline src/docs/getting-started/quick-start.md:101]: Running sealer  image as a Non-root User

File: src/docs/advanced/use-sealer-rootless.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
+# Run sealer in non-root mode
+
+## Install sealer using sudo
+
+**Resolved remote read permission to `sshd_config` file:**
```
Comment (kakaZhou719) [inline src/docs/advanced/use-sealer-rootless.md:5]: do you mean we need to check the permission of all nodes? if not 644, need chmod? if so,we can describe it more clearly .

File: src/docs/advanced/use-sealer-rootless.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+
+
+### Ensure all nodes participating in the cluster have passwordless login permissions
+
+**Still needs to assign non-secret permissions to user sudo(Every node except master0):**
+
+You need to give passwordless login permissions to users on all nodes: `vim /etc/sudoers`. It is a configuration file that contains sudo user and group rules, allowing a specific user (sealer) or group to run commands as a superuser when executing commands.
+
+```bash
+sealer ALL=(ALL) NOPASSWD: NOPASSWD: ALL
```
Comment (kakaZhou719) [inline src/docs/advanced/use-sealer-rootless.md:20]: is this the rules that we need to make sure it is meet our expectation ? we could describe it more clearly .

File: src/docs/advanced/use-sealer-rootless.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
+# Run sealer in non-root mode
+
+## Install sealer using sudo
```
Comment (kakaZhou719) [inline src/docs/advanced/use-sealer-rootless.md:3]: here is my suggestion :)
```
## prerequisites 
### check `sshd_config` permission
### check `sudoers` rules
## run sealer image
```

File: src/docs/advanced/use-sealer-rootless.md (hunk 2 above)
Comment (cubxxw) [reply src/docs/advanced/use-sealer-rootless.md:5]: yes,Only the current running node needs to execute local sshd permissions.So I didn't tell them to configure on all nodes

File: src/docs/advanced/use-sealer-rootless.md (hunk 3 above)
Comment (cubxxw) [reply src/docs/advanced/use-sealer-rootless.md:20]: yes, but this one seems very mature and I think very detailed

File: src/docs/advanced/use-sealer-rootless.md (hunk 4 above)
Comment (cubxxw) [reply src/docs/advanced/use-sealer-rootless.md:3]: Ok, I'll add that later

File: src/docs/getting-started/quick-start.md [hunk 5]
```diff
@@ -18,6 +18,9 @@ sealer run docker.io/sealerio/kubernetes:v1.22.15 \
   --masters 192.168.0.2,192.168.0.3,192.168.0.4 \
   --nodes 192.168.0.5,192.168.0.6,192.168.0.7 --passwd xxx
 ```
+:::tip Password reminder
```
Comment (kakaZhou719) [inline src/docs/getting-started/quick-start.md:21]: nice optimization make our quick start more readable . 👍

**Comment by kakaZhou719** [comment]
image
pls fix lint

**Pull request closed without merging**